README.md

```diff
@@ -1,11 +1,11 @@
 # rpi-eeprom
-This repository contains the scripts and pre-compiled binaries used to create the `rpi-eeprom` package which is used to update the Raspberry Pi 4 bootloader and VLI USB xHCI controller EEPROMs.
+This repository contains the scripts and pre-compiled binaries used to create the `rpi-eeprom` package which is used to update the Raspberry Pi 4B, 400 and CM4 bootloader and VLI USB xHCI controller EEPROMs.
```

Comment:
This change _might_ be misinterpreted to mean that there's separate EEPROM packages for Raspberry Pi 4B, 400 and CM4? :shrug:

Reply:
Suggestion. Can we agree on shorthand for Raspberry Pi 4B, CM4 and Pi 400? This appears in multiple places in the docs and feels unwieldy.

Reply:
ping @JamesH65 for his thoughts.

Reply:
Oh, and @aallan too :slightly_smiling_face:

Reply:
> Suggestion. Can we agree on shorthand for Raspberry Pi 4B, CM4 and Pi 400? This appears in multiple places in the docs and feels unwieldy.

That would be very handy. "Raspberry Pi 4-series" would be one possibility; "BCM2711-based Raspberry Pi computers" another. I think if there was an easy answer to this, someone would have come up with a suitable shorthand already 😮

Reply:
Pi 4 family or just Pi 4 (without the B).

Reply:
OK, I've reverted my change to that line - Raspberry Pi 4 it is 👍

Reply:
Using "Pi 4" or "Raspberry Pi 4" to refer to the entire BCM2711 family seems like a good idea to me.